h2/config.py

```diff
@@ -25,6 +26,21 @@ def __set__(self, instance, value):
         setattr(instance, self.attr_name, value)
 
 
+class DummyLogger(logging.Logger):
```

Comment:
Ok, so this is really good, and I'm sorry this popped up so late, but:

Let's not inherit from `logging.Logger`. Let's just write a class that conforms to the minimal interface of it. So, we add an `__init__` that just takes `*args`.

That way we avoid any risk of that object changing.